internal/telemetry/telemetry.go

```diff
@@ -115,11 +116,7 @@ func (t *Telemetry) Finish(event string, cmdError error) {
 	}
 	defer telemetryClient.Close()
 
-	err = telemetryClient.Upload(t.telemetryData)
-	if err != nil {
-		t.factory.Logger.Info("Cannot send data to telemetry: %q", err)
-	}
-
+	_ = telemetryClient.Upload(t.telemetryData)
```

Comment:
Should we still log the error in verbose mode?

Reply:
Good question. I will add that to debug